### PR TITLE
collapse changes with <details></details>

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Fix https://github.com/containerd/containerd/issues/4453

Example: `release-tool --dry  --tag v1.4.0-rc.X --linkify releases/v1.4.0-rc.toml`↓

---

containerd 1.4.0-rc.X

Welcome to the v1.4.0-rc.X release of containerd!  
*This is a pre-release of containerd*
...

### Changes from containerd/zfs
<details><summary>3 commits</summary>
<p>

* [`9abf673`](https://github.com/containerd/zfs/commit/9abf673ca6ff9ab8d9bd776a4ceff8f6dc699c3d) Merge pull request  [#25](https://github.com/containerd/zfs/pull/25) from Zyqsempai/fix-walk-method-interface
* [`73767d7`](https://github.com/containerd/zfs/commit/73767d7cc85aa4bdb2bca980ac884ef3fd5bfb75) Added filters to Walk method signature
* [`456dcfd`](https://github.com/containerd/zfs/commit/456dcfd8803bbaabd31a91d346ce7e37af7a5053) Fix snapshot commit to pass opts to storage
</p>
</details>

...

### CODE REVIEW

File: template.go [hunk 1]
```diff
@@ -45,9 +45,13 @@ https://github.com/{{.GithubRepo}}/issues.
 {{range $project := .Changes}}
 
 ### Changes{{if $project.Name}} from {{$project.Name}}{{end}}
+<details><summary>{{len $project.Changes}} commits</summary>
```
Comment (dmcgowan) [inline template.go:48]: if `len` is 1, no `s` on commit

**Comment by dmcgowan** [comment]
We use the same output for our git tags, but I agree this is a change we should make so our release notes a bit more scrollable

File: template.go (hunk 1 above)
Comment (AkihiroSuda) [reply template.go:48]: Won't be 1 because we have merge commits

Comment (dmcgowan) [reply template.go:48]: I agree, but still better not to make that assumption for all projects which use this tool.

Comment (AkihiroSuda) [reply template.go]: updated

**Pull request merged**